examples/remix/README.md

```diff
@@ -0,0 +1,186 @@
+# Use with Remix
+
+This example demonstrates how to use `jest-preview` with [`Remix`](https://remix.run/), bootstrapped using [`create-remix`](https://remix.run/docs/en/v1#getting-started) (without using any built-in templates/stacks).
+
+🚧 This example only demonstrates the simplest use of CSS in a Remix app (global stylesheet). Remix supports [many different ways of styling](https://remix.run/docs/en/v1/guides/styling). Example of more complex ways to style is coming.
+
+## Install Jest
+
+You have to manually install Jest in your Remix app.
+
+First, install `jest` itself. If you are using `jest` version 28 or later, you must install `jest-environment-jsdom` too ([reference](https://jestjs.io/docs/upgrading-to-jest28#jsdom)).
+
+```bash
+npm install --save-dev jest jest-environment-jsdom
+```
+
+Install Babel
+
+```bash
+npm install --save-dev @babel/preset-env @babel/preset-react babel-jest
+```
+
+Config Babel inside `babel.config.js`
+
+```bash
+module.exports = {
+  presets: [
+    ['@babel/preset-env', { targets: { node: 'current' } }],
+    ['@babel/preset-react', { runtime: 'automatic' }],
+  ],
+};
+```
```

Comment:
Since `jest-preview` is advocating for @swc-project (because of the speed gain). I think we should setup Jest with https://github.com/swc-project/jest. Refer to https://github.com/nvh95/jest-preview/blob/f0ef957bf89cbe9f0f94389ccf37d33e53b8a6f8/.swcrc and https://github.com/nvh95/jest-preview/blob/30ba86e1d6d74b897bf121837ccf9088afe4395d/jest.config.js#L14 for more